<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LuxWidget</class>
 <widget class="QWidget" name="LuxWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>Select your main *.lxs file</string>
       </property>
       <property name="text">
        <string>Main scene file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="mainSceneFileLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="chooseMainSceneFileButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>Resolution (width x height) of a final image</string>
       </property>
       <property name="text">
        <string>Output resolution</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="outputResXSpinBox">
         <property name="maximum">
          <number>1000000000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="outputResYSpinBox">
         <property name="maximum">
          <number>1000000000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string>In which format would you like to recieve the result</string>
       </property>
       <property name="text">
        <string>Output format</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="outputFormatsComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string>Where to save the result</string>
       </property>
       <property name="text">
        <string>Output file</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="outputFileLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="chooseOutputFileButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Subtask stop conditions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="stopByTimeRadioButton">
          <property name="toolTip">
           <string>Stop subtask computation after the given number of seconds</string>
          </property>
          <property name="text">
           <string>Halt time</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="stopBySppRadioButton">
          <property name="toolTip">
           <string>Stop subtask computation when average number of samples per pixel is higher or equal than the given value</string>
          </property>
          <property name="text">
           <string>Halt spp</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="haltTimeLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="haltSppLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
